assets/css/layout/_layout.scss

```diff
@@ -1,16 +1,30 @@
 // Layout styles.
 @import "header";
 
-article {
+body {
+  background-attachment: fixed;
+  background-image: url("../images/WeCount website background_distribution.svg");
```

Comment:
Could you rename the filename of the background image to `background.svg`? Thanks.